glean-core/ios/Glean/Glean.swift

```diff
@@ -28,6 +28,7 @@ public class Glean {
     var initialized: Bool = false
     private var uploadEnabled: Bool = true
     private var debugViewTag: String?
+    var logPings: Bool?
```

Comment:
This could also be given a default value of false rather than being an Optional.